index.html

```diff
@@ -179,6 +196,67 @@ <h3>
           }
         </pre>
       </section>
+      <section>
+        <h3>
+          Processing the `note_taking` member
+        </h3>
+        <p>
+          To <dfn>process the `note_taking` member</dfn>, given [=ordered map=]
+          |json:ordered map|, [=ordered map=] |manifest:ordered map|, [=URL=]
+          |manifest_URL:URL|, run the following during the
+          <a data-cite=
+          "appmanifest#dfn-processing-extension-point-of-web-manifest">extension
+          point</a> in [=processing a manifest=]:
+        </p>
+        <ol class="algorithm">
+          <li>If |json|["note_taking"] does not [=map/exist=], return.</li>
+          <li>If the type of |json|["note_taking"] is not [=ordered map=],
+            return.
+          </li>
+          <li>Set |manifest|["note_taking"] to a new [=ordered map=].</li>
+          <li>If |json|["note_taking"]["new_note_url"] does not [=map/exist=],
+            return.
+          </li>
+          <li>If the type of |json|["note_taking"]["new_note_url"] is not
+            [=string=], return.
+          </li>
+          <li>Let |new_note_url:URL| be the result of [=URL Parser|parsing=]
+            |json|["note_taking"]["new_note_url"] with |manifest URL| as the
+            base URL.
+          </li>
+          <li>If |new_note_url| is failure, return.</li>
+          <li>If |new_note_url| is not [=manifest/within scope=] of |manifest
+            URL|, return.
+          </li>
+          <li>Set |manifest|["note_taking"]["new_note_url"] to |new_note_url|.
+          </li>
+        </ol>
+      </section>
+      <section>
+        <h3>
+          Launching the `new_note_url`
+        </h3>
+        <p>
+          To <dfn>launch the `new_note_url`</dfn>, given <a data-cite=
+          "appmanifest#dfn-processed-manifest">processed manifest</a>
+          |manifest:processed manifest|, run the following steps:
+        </p>
+        <ol>
+          <li>If |manifest|["note_taking"] does not [=map/exist=], return.</li>
+          <li>If |manifest|["note_taking"]["new_note_url"] does not
+            [=map/exist=], return.
+          </li>
+          <li>If the type of |manifest|["note_taking"]["new_note_url"] is not
+            [=URL=], return.
+          </li>
+          <li>Let |browsing context:Browsing Context| be the result of creating
+            a new [=top-level browsing context=].
+          </li>
+          <li>[=Navigate=] |browsing context| to resource
```

Comment:
Another aside: I suspect all of these "create a new browsing context / navigate to URL" will need to change once we have the launch handler stuff in the spec.

So we should probably create a function called "launch a URL" which invokes that mechanism. Again, something for later but just writing it down.

Reply:
Agreed.

Reply:
https://github.com/w3c/manifest/issues/986